crates/bevy_ecs/src/entity/mod.rs

```diff
@@ -243,6 +243,10 @@ pub struct Entities {
 }
 
 impl Entities {
+    pub fn meta_len(&self) -> usize {
```

Comment:
Please leave a doc comment here. Otherwise is just a dangling random `len` accessor.